modules/nlu/src/backend/election/natural-election.test.ts

```diff
@@ -0,0 +1,48 @@
+import { extractElectedIntentSlot } from './natural-election'
+import { fakeEventUnderstanding } from '../../test/types'
+
+describe('naturalElection', () => {
```

Comment:
Can you try adding `"import '../../../../../packages/bp/src/sdk/botpress'"` at the top of the file ? It's ugly, I know, but I believe 4 / 6 of these tests won't build when the sdk is properly resolved. 

![image](https://user-images.githubusercontent.com/25970722/135161002-9dc1ade2-720e-40d8-9b94-3dcf3fd4f246.png)

I personally don't like testing code with invalid typings.

Reply:
Done